lib/PromoteStackPointer/SegregateStackAccessesPass.cpp

```diff
@@ -529,7 +523,7 @@ class SegregateStackAccesses {
         break;
 
       case ReturnMethod::ModelAggregate:
-        ResultVariableType = Layout.returnValueAggregateType();
+        // Nothing to check here.
```

Comment:
The `upgradeLocalFunctions` seems to have some semantics changes. Can you describe them?

Reply:
Ping

Reply:
Ah, sorry, it seems like I missed this one.

What do you mean? The only non-trivial change there is one variable being cut since it's only even used in one place inside the `if (SameConditionAsWhenItWasSet())`, so there's no reason not to "assign" it there directly.